tox.ini

```diff
@@ -29,7 +29,7 @@ commands = flake8 --enable-extensions=FS003
 # Set flake8 configuration options which are used by the `flake8` command in [testenv:pep8]
 [flake8]
 # W504 ignore wrap line after binary operator
-ignore = W504
+ignore = W504, F405, F403
```

Comment:
Whoops maybe explicit imports no-matter-what are a better idea after all... it's about flake8 here, but will mypy be able to distinguish b/w undefined identifier and a star-imported one?

In fact, I'd revert this change to tox.ini, use explicit imports everywhere on this PR, and discuss within the scope of #740